app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsFragment.kt

```diff
@@ -70,7 +70,7 @@ class TransactionDetailsFragment : BaseViewBindingFragment<FragmentTransactionDe
                 Navigation.findNavController(root).navigateUp()
             }
             refresh.setOnRefreshListener {
-                viewModel.loadDetails(txId)
+                viewModel.loadDetails(viewModel.txDetails?.txId ?: txId)
```

Comment:
Please add comment explaining the reasoning behind loading details with id. Better would be to introduce additional nav arg called txHash and clearly distinguish btw 2 cases.

Reply:
Aas far as I could tell, we do not have the txHash at that point. Because as I understand we only get it after the tx has been proposed. But maybe I am wrong.

Reply:
you mean txId? It would be helpful to have this explanation in a comment.

Reply:
Yes, I meant to write tzxId